src/main/java/it/gov/pagopa/payhub/activities/dto/paymentsreporting/PaymentsReportingDTO.java

```diff
@@ -0,0 +1,46 @@
+package it.gov.pagopa.payhub.activities.dto.paymentsreporting;
+
+import it.gov.pagopa.payhub.activities.dto.IngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.OrganizationDTO;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.sql.Timestamp;
+import java.util.Date;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class PaymentsReportingDTO {
+
+	private int version;
+	private Timestamp creationDate;
+	private Timestamp lastChangeDate;
+	private OrganizationDTO orgId;
+	private IngestionFlowFileDTO ingestionFlowFile;
+	private String idPsp;
+	private String flowIdentifierCode;
+	private Date flowDateTime;
```

Comment:
let's use new date/time class
```suggestion
	private LocalDateTime flowDateTime;
```

Reply:
This is a LocalDate or a LocalDateTime? I suggested LocalDateTime based on field name, but you should check each field if should contains also the time or just the date